src/common/snippets/src/op/subgraph.cpp

```diff
@@ -428,6 +429,7 @@ void Subgraph::data_flow_transformations(const BlockedShapeVector& blocked_input
     manager.register_pass<snippets::pass::ConvertConstantsToScalars>();
 
     manager.register_positioned_passes(backend_passes);
+    manager.register_pass<snippets::pass::TransformConvertToConvertTruncation>();
```

Comment:
We call this transformation in [`CommonOptimizations`](https://github.com/openvinotoolkit/openvino/blob/158d71f5d01a8b584cc2e715bb2a1e5c5712f28f/src/common/snippets/src/pass/common_optimizations.cpp#L49) after subgraph tokenization to mark all original `Convert` ops as `ConvertTruncation`.

Then all other transformations which inserts `Convert` op (such as `FQDecomposition` or `PropagatePrecision`), should insert utility `ConvertSaturation` op.

Did you find the pass after `TransformConvertToConvertTruncation` registration in `CommonOptimizations` which inserts clear `Convert` op (not from Snippets dialect)?

Reply:
I can't reproduce the issue locally. I checked the passes after ConvertTruncation in commonOptimization. Only found an potential possible [convert ](https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/constant_fold_utils.cpp#L143)in [ConstantFolding](https://github.com/openvinotoolkit/openvino/blob/master/src/common/snippets/src/op/subgraph.cpp#L427), but not very sure. So I need more detailed info from the issue reporter:-)